packages/console/src/pages/TenantSettings/TenantBasicSettings/index.tsx

```diff
@@ -72,6 +78,34 @@ function TenantBasicSettings() {
     await saveData({ name, tag });
   });
 
+  const onClickDeletionButton = () => {
+    setIsDeletionModalOpen(true);
+  };
+
+  const onDelete = async () => {
+    if (isDeleting) {
+      return;
+    }
+
+    setIsDeleting(true);
+    try {
+      await cloudApi.delete(`/api/tenants/${currentTenantId}`);
+      setIsDeletionModalOpen(false);
+      void mutate();
```

Comment:
Need to await? Or the next line may not work as expected

Reply:
Maybe not. Mutate seems has nothing to do with the following code

Reply:
I think we should use await, since we may delete the last tenant and hence should direct user to create tenant landing page (not ready yet).